data/validations/sharedValidations.js

```diff
@@ -1,16 +1,14 @@
-function checkArgumentLength(args, length) {
-    let argLength = args.length;
```

Comment:
We may want to keep this so that you dont have to pass in `arguments.length` every time